jsonp/2.0/_index.md

```diff
@@ -1,27 +1,26 @@
 ---
-title: "Jakarta JSON Processing 2.0 (under development)"
-date: 2020-01-15
+title: "Jakarta JSON Processing 2.0 (milestone)"
+date: 2020-05-31
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta JSON Processing defines a Java(R) based framework for parsing, generating, transforming, and
 querying JSON documents.
 
-* [Jakarta JSON Processing 2.0 Release Record]()
+* [Jakarta JSON Processing 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jsonp/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta JSON Processing 2.0 Specification Document]() (PDF)
-* [Jakarta JSON Processing 2.0 Specification Document]() (HTML)
 * [Jakarta JSON Processing 2.0 Javadoc](./apidocs)
 * [Jakarta JSON Processing 2.0 TCK]()([sig](),[sha](),[pub]())
 * Maven coordinates
-  * [jakarta.jsonp:jakarta.jsonp-api:jar:2.0.0]()
-
+  * [jakarta.jsonp:jakarta.jsonp-api:jar:2.0.0-RC2](https://repo1.maven.org/maven2/jakarta/json/jakarta.json-api/2.0.0-RC2/jakarta.json-api-2.0.0-RC2.jar)
 
 # Compatible Implementations
 
-* [Jakarta JSON Processing]()
+* [Jakarta JSON Processing](https://repo1.maven.org/maven2/org/glassfish/jakarta.json/2.0.0-RC2/jakarta.json-2.0.0-RC2.jar)
 
 # Ballots
 
+## Release Review
```

Comment:
You already have the "Release Review" section later in this _index.md file, there is no need to repeat it.  And, it should be after the Plan Review section anyway (where it currently exists).